exchanges/okx/okx.go

```diff
@@ -2158,42 +2158,36 @@ func (ok *Okx) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (flo
 	var fee float64
 	switch feeBuilder.FeeType {
 	case exchange.CryptocurrencyTradeFee:
-		var responses []TradeFeeRate
 		uly, err := ok.GetUnderlying(feeBuilder.Pair, asset.Spot)
 		if err != nil {
 			return 0, err
 		}
-		responses, err = ok.GetTradeFee(ctx, okxInstTypeSpot, uly, "")
+		responses, err := ok.GetTradeFee(ctx, okxInstTypeSpot, uly, "")
 		if err != nil {
 			return 0, err
 		} else if len(responses) == 0 {
 			return 0, errors.New("no trade fee response found")
 		}
 		if feeBuilder.IsMaker {
-			if fee, err = strconv.ParseFloat(responses[0].FeeRateMaker, 64); err != nil || fee == 0 {
-				fee, err = strconv.ParseFloat(responses[0].FeeRateMakerUSDT, 64)
-				if err != nil {
-					return fee, err
-				}
+			if feeBuilder.Pair.Quote.Equal(currency.USDC) {
+				fee = responses[0].FeeRateMakerUSDC.Float64()
+			} else if fee = responses[0].FeeRateMaker.Float64(); fee == 0 {
```

Comment:
I missed the setting in the if statement and couldn't see how fee was being set for LTC-ETH for too long 😭